packages/wallets/src/features/accounts/screens/ResubmitPOA/ResubmitPOA.scss

```diff
@@ -0,0 +1,83 @@
+.wallets-resubmit-poa {
+    width: 99.6rem;
+    height: 60rem;
+    display: flex;
+    padding: 1.6rem;
+    flex-direction: column;
+    align-items: center;
+    gap: 1.6rem;
+    flex: 1 0 0;
+    align-self: stretch;
+    background: var(--system-light-8-primary-background, #fff);
+
+    @include mobile {
+        width: 100%;
+        height: 100%;
+        padding: 1.6rem;
+    }
+
+    &__address {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 1.6rem;
+        max-width: 85rem;
+
+        @include mobile {
+            width: 100%;
+        }
+
+        &__title {
```

Comment:
BEM recommends to avoid block nesting (in order to limit complexity)

Reply:
so e.g.
.wallets-resubmit-poa
.wallets-resubmit-poa__adress
.wallets-resubmit-poa__adress-title